CMakeLists.txt

```diff
@@ -76,6 +67,14 @@ else()
   set(PAL az_noplatform)
 endif()
 
+add_subdirectory(sdk/core/az_core)
```

Comment:
**Note**: this is reordered so that the `${PAL}` target can be linked to in the sub-directories here and below.

Reply:
Add that as a comment in the file itself.